docs/site/Controllers.md

```diff
@@ -122,104 +148,125 @@ The `@api` decorator allows you to annotate your Controller with a specification
   }
 }
 ```
 
-- `HelloRepository` extends from `Repository`, which is LoopBack's database abstraction. See [Repositories](./Repositories.md) for more.
-- `HelloMessage` is the arbitrary object that `list` returns a list of.
-- `@get('/messages')` creates the `Route` for the Operation using `app.route()`.
-- `@param.query.number` adds a `number` param with a source of `query`.
+* `HelloRepository` extends from `Repository`, which is LoopBack's database
+  abstraction. See [Repositories](./Repositories.md) for more.
+* `HelloMessage` is the arbitrary object that `list` returns a list of.
+* `@get('/messages')` automatically creates the
+  [Paths Item Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#path-item-object)
+  for OpenAPI spec, which also handles request routing.
+* `@param.query.number` specifies in the spec being generated that the route
+  takes a parameter via query which will be a number.
 
 ## Handling Errors in Controllers
 
-In order to specify errors for controller methods to throw, the class `HttpErrors` is used. `HttpErrors` is a class that has been re-exported from [http-errors](https://www.npmjs.com/package/http-errors), and can be found in the `@loopback/rest` package.
-
-Listed below are some of the most common error codes. The full list of supported codes is found [here](https://github.com/jshttp/http-errors#list-of-all-constructors).
-
-|Status Code|Error                |
-|-----------|---------------------|
-|400        |BadRequest           |
-|401        |Unauthorized         |
-|403        |Forbidden            |
-|404        |NotFound             |
-|500        |InternalServerError  |
-|502        |BadGateway           |
-|503        |ServiceUnavailable   |
-|504        |GatewayTimeout       |
-
-The example below shows the previous controller revamped with `HttpErrors` along with a test to verify that the error is thrown properly.
+In order to specify errors for controller methods to throw, the class
+`HttpErrors` is used. `HttpErrors` is a class that has been re-exported from
+[http-errors](https://www.npmjs.com/package/http-errors), and can be found in
+the `@loopback/rest` package.
+
+Listed below are some of the most common error codes. The full list of supported
+codes is found
+[here](https://github.com/jshttp/http-errors#list-of-all-constructors).
+
+| Status Code | Error               |
+| ----------- | ------------------- |
+| 400         | BadRequest          |
+| 401         | Unauthorized        |
+| 403         | Forbidden           |
+| 404         | NotFound            |
+| 500         | InternalServerError |
+| 502         | BadGateway          |
+| 503         | ServiceUnavailable  |
+| 504         | GatewayTimeout      |
+
+The example below shows the previous controller revamped with `HttpErrors` along
+with a test to verify that the error is thrown properly.
 
 ```js
 // the test
```

Comment:
Show the test file name to help make sense of the relative paths below. Example: `/test/unit/controllers`